Create ui-c8y-1023-64-1-added-a-fix-for-preview-not-working-when-removing-and-re.md (#4478)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-64-1-added-a-fix-for-preview-not-working-when-removing-and-re.md

---
date: ""

> title: Fixed preview not working when removing and re-adding data points

Reviewer: ```suggestion
title: Widget configuration preview now properly shows data points that have been removed and re-added
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65370
version: 1023.64.1
---

> When you removed a data point from the data graph widget configuration and re-added it, the preview would not work as expected. This issue has been fixed so that the preview now properly shows data points after they are removed and re-added, allowing you to work with your data point configurations without encountering display issues.

Reviewer: ```suggestion
When removing a data point from the "Data graph" widget configuration and re-adding it, the preview did not work as expected. This issue has been fixed so that the preview now properly shows data points that have been removed and re-added, allowing you to work with your data point configurations without encountering display issues.
```
